readme.txt

```diff
@@ -20,18 +20,18 @@ For installation instructions, visit [docs.pressbooks.org/installation](https://
 TK.
 
 == Changelog ==
-= 5.15.0=
+= 5.15.1=
 
-* See: https://github.com/pressbooks/pressbooks/releases/tag/5.15.0
+* See: https://github.com/pressbooks/pressbooks/releases/tag/5.15.1
 * Full release history available at: https://github.com/pressbooks/pressbooks/releases
 
 == Upgrade Notice ==
-= 5.15.0 =
+= 5.15.1 =
 
-* Pressbooks 5.15.0 requires PHP >= 7.1.
-* Pressbooks 5.15.0 requires [WordPress 5.4](https://wordpress.org/support/wordpress-version/version-5-3/)
-* Pressbooks 5.15.0 requires [McLuhan >= 2.10.2](https://github.com/pressbooks/pressbooks-book/)
-* Pressbooks 5.15.0 supports integration with [Sentry](https://sentry.io/)
+* Pressbooks 5.15.1 requires PHP >= 7.1.
+* Pressbooks 5.15.1 requires [WordPress 5.4](https://wordpress.org/support/wordpress-version/version-5-3/)
+* Pressbooks 5.15.1 requires [McLuhan >= 2.10.2](https://github.com/pressbooks/pressbooks-book/)
+* Pressbooks 5.15.1 supports integration with [Sentry](https://sentry.io/)

```

Comment:
Didn't we add support for Sentry to 5.15.0? https://github.com/pressbooks/pressbooks/releases/tag/5.15.0

Reply:
Nevermind. I understand the purpose of the note. Ignore suggestion.